test/Dockerfile-hsm

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:15.10
```

Comment:
Why Ubuntu? The golang image, and therefore boulder-tools and boulder, are based on buildpack-deps:jessie-scm, which I think is somewhat smaller (but more importantly people are likely to have it).